File: tracking-service-master/tracking-web/src/app/model/product.ts
export class Product {
    _id: number;
    _name: string;
    _sku: string;
    _barcode: string;
    _image: string;
    _price: string;
    _enabled: boolean;

    public get id(): number {
        return this._id;
    }

    public set id(id: number) {
        this._id = id;
    }

    public get name(): string {
        return this._name;
    }

    public set name(name: string) {
        this._name = name;
    }

    public get sku(): string {
        return this._sku;
    }

    public set sku(sku: string) {
        this._sku = sku;
    }

    public get barcode(): string {
        return this._barcode;
    }

    public set barcode(barcode: string) {
        this._barcode = barcode;
    }

    public get image(): string {
        return this._image;
    }

    public set image(image: string) {
        this._image = image;
    }

    public get price(): string {
        return this._price;
    }

    public set price(price: string) {
        this._price = price;
    }

    public get enabled(): boolean {
        return this._enabled;
    }

    public set enabled(enabled: boolean) {
        this._enabled = enabled;
    }
}

File: tracking-service-master/tracking-server/src/main/java/com/technicaltest/trackingserver/model/Product.java
package com.technicaltest.trackingserver.model;

import com.fasterxml.jackson.annotation.JsonProperty;

public class Product {

    @JsonProperty("_id")
    private Long id;
    @JsonProperty("_name")
    private String name;
    @JsonProperty("_sku")
    private String sku;
    @JsonProperty("_barcode")
    private String barcode;
    @JsonProperty("_image")
    private String image;
    @JsonProperty("_price")
    private String price;
    @JsonProperty("_enabled")
    private Boolean enabled;

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getSku() {
        return sku;
    }

    public void setSku(String sku) {
        this.sku = sku;
    }

    public String getBarcode() {
        return barcode;
    }

    public void setBarcode(String barcode) {
        this.barcode = barcode;
    }

    public String getImage() {
        return image;
    }

    public void setImage(String image) {
        this.image = image;
    }

    public String getPrice() {
        return price;
    }

    public void setPrice(String price) {
        this.price = price;
    }

    public Boolean getEnabled() {
        return enabled;
    }

    public void setEnabled(Boolean enabled) {
        this.enabled = enabled;
    }
}
